### PR TITLE
Update flutter_command.dart

### PR DESCRIPTION
**Pull request by amanv8060**
removed absolute path . More description in https://github.com/flutter/website/pull/4607

## Description

*Replace this paragraph with a description of what this PR is doing. If you're modifying existing behavior, describe the existing behavior, how this PR is changing it, and what motivated the change. If you're changing visual properties, consider including before/after screenshots (and runnable code snippets to reproduce them).*

## Related Issues

https://github.com/flutter/website/issues/4608

## Tests

None As it is a just a valueHint Change

## Checklist

- [x] I read the [Contributor Guide] and followed the process outlined there for submitting PRs.
- [x] I signed the [CLA].
- [x] I read and followed the [Flutter Style Guide], including [Features we expect every widget to implement].
- [x] I read the [Tree Hygiene] wiki page, which explains my responsibilities.
- [x] I updated/added relevant documentation (doc comments with `///`).
- [x] All existing and new tests are passing.
- [x] The analyzer (`flutter analyze --flutter-repo`) does not report any problems on my PR.
- [x] I am willing to follow-up on review comments in a timely manner.

## Breaking Change

Did any tests fail when you ran them? Please read [Handling breaking changes].

- [x] No, no existing tests failed, so this is *not* a breaking change.
- [ ] Yes, this is a breaking change. *If not, delete the remainder of this section.*
   - [ ] I wrote a design doc: https://flutter.dev/go/template *Replace this with a link to your design doc's short link*
   - [ ] I got input from the developer relations team, specifically from: *Replace with the names of who gave advice*
   - [ ] I wrote a migration guide: https://flutter.dev/go/breaking-changes-template *Replace this with a link to a pull request that adds the migration guide to https://flutter.dev/docs/release/breaking-changes*

[issue database]: https://github.com/flutter/flutter/issues
[Contributor Guide]: https://github.com/flutter/flutter/wiki/Tree-hygiene#overview
[Test Coverage]: https://github.com/flutter/flutter/wiki/Test-coverage-for-package%3Aflutter
[Flutter Style Guide]: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo
[Features we expect every widget to implement]: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#features-we-expect-every-widget-to-implement
[CLA]: https://cla.developers.google.com/
[Tree Hygiene]: https://github.com/flutter/flutter/wiki/Tree-hygiene
[Handling breaking changes]: https://github.com/flutter/flutter/wiki/Tree-hygiene#handling-breaking-changes

### CODE REVIEW

File: packages/flutter_tools/lib/src/runner/flutter_command.dart [hunk 1]
```diff
@@ -459,7 +459,7 @@ abstract class FlutterCommand extends Command<void> {
         'the information needed to symbolize Dart stack traces. For an app built '
         "with this flag, the 'flutter symbolize' command with the right program "
         'symbol file is required to obtain a human readable stack trace.',
-      valueHelp: '/project-name/v1.2.3/',
+      valueHelp: '../project-name/v1.2.3/',
```
Comment (jonahwilliams) [inline packages/flutter_tools/lib/src/runner/flutter_command.dart:462]: just remove `../project-name ` entirely, here and below

File: packages/flutter_tools/lib/src/runner/flutter_command.dart [hunk 2]
```diff
@@ -486,7 +486,7 @@ abstract class FlutterCommand extends Command<void> {
         'during "flutter run". These can be included in an application to '
         'improve the first frame render times.',
       hide: hide,
-      valueHelp: '/project-name/flutter_1.sksl'
+      valueHelp: '../project-name/flutter_1.sksl '
```
Comment (jonahwilliams) [inline packages/flutter_tools/lib/src/runner/flutter_command.dart:489]: also remove trailing space

File: packages/flutter_tools/lib/src/runner/flutter_command.dart (hunk 1 above)
Comment (amanv8060) [reply packages/flutter_tools/lib/src/runner/flutter_command.dart:462]: ok! , i will do it

**Review by jonahwilliams — APPROVED** [review]
LGTM

Lack of tests is fine, its just an update to documentation (essentially)

**Comment by jonahwilliams** [comment]
(PR will be automatically submitted when the tree is green)

**Pull request merged**